amigos/amigos/gpio.py

```diff
@@ -136,8 +131,69 @@ def power_up(bit):
     if bit:
         __toggle(bit-1)
         subprocess.call("echo 0xFF> /sys/class/gpio/pwr_ctl/data", shell=True)
-        sleep(2)
         __toggle(bit)
         subprocess.call("echo 0xFF > /sys/class/gpio/pwr_ctl/data", shell=True)
-        print("Power is up on peripherals")
-    __update_bit('0b11111111,1b11111111')
+        print("ok")
+        __update_bit('0b11111111,0b11111111')
+
+
+def CR1000_on(bit):
+    """
+    Turn the power off gps module on after toggling the bit
```

Comment:
This section was copied and paste. So will be fixed by the appropriate person tomorrow